### ISSUE TITLE
Link with `nnet`

### ISSUE THREAD

**Issue by NRaillard** [ejikeugba/gofcat#3]
Hello,

I am still conducting tests for the JOSS review (openjournals/joss-reviews#4382).
I was testing the use of a `nnet` fitted model. I just took the example from the help of the function, which is a binary  response model but I think it should work ? 

```r
library(nnet)
library(MASS)
example(birthwt)
bwt.mu <- multinom(low ~ ., bwt)
erroR(bwt.mu)
Rsquared(bwt.mu)
```
I obtain the following error: 

> Error: more than one column of fitted values for a binary model is not allowed

**Comment by ejikeugba** [comment]
Thanks for spotting it! the bug is now fixed with [e95b9db](https://github.com/ejikeugba/gofcat/commit/e95b9db4a622ee71d55ed31f586f91294047ef9f)

**Comment by NRaillard** [comment]
Well done :)
